src/datachain/lib/dc/datachain.py

```diff
@@ -919,9 +937,20 @@ def batch_map(
             ```
         """
         udf_obj = self._udf_to_obj(BatchMapper, func, params, output, signal_map)
+        chunk_rows = self._settings.chunk_rows
```

Comment:
we are mixing up `chunk` and user facing `batch` size - as I commented before. The chage should not affect `batch_map()` in any way.